recipes/pytentiostat/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "pytentiostat" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/juami/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 952e6e2743d88a3c519a9644f5514902f4b1726f67221727c71b7a37dd59ea5f
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python >=3
+    - setuptools
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3
+    - xonsh
+    - pyfirmata
+    - pandas
+    - matplotlib
+    - pyyaml
+
+test:
+  requires:
+    - pytest
+  imports: []
+
+about:
+  home: https://github.com/juami/pytentiostat
+  license: BSD 3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'python code to run the JUAMI potentiostat'
+
+  description: |
+    The JUAMI potentiostat is a cheap (<$100) potentiostat built on the Arduino platform
+    for use in classroom demonstrations and simple research.  It can carry out
+    electrochemical scans such as one would use for testing performance of an
+    electrochemical cell or battery.  It was developed by a team from JUAMI, the
+    joint undertaking for an African Materials Institute (www.juami.org) which has
+    a mission to help develop research collaborations in materials science between
+    US and African researchers.  Pytentiostat is python code for running the
+    potentiostat.  For more information, contact Prof. Simon Billinge at email
+    sb2896@columbia.edu
+  doc_url: https://pytentiostat.github.io/
+  dev_url: https://github.com/juami/pytentiostat
+
+extra:
+  recipe-maintainers:
+    - sbillinge
+    - chiahaoliu
```

Comment:
@chiahaoliu - are you OK with being a maintainer?

Reply:
yes, I am